tests/visual-tests/playground/Toolbar.visual.test.tsx

```diff
@@ -0,0 +1,40 @@
+import {test} from 'playwright/core';
+
+import {Playground} from './Playground.helpers';
+
+test.describe('Toolbar', () => {
+    test.beforeEach(async ({mount}) => {
+        await mount(<Playground initial="" />, {
+            hidePlaygroundBlocks: true,
+            width: '100%',
+            rootStyle: {
+                width: '100%',
+                height: '100%',
+                padding: 0,
+            },
+        });
+    });
+
+    test('should hide actions at small width', async ({
```

Comment:
Suggest testing both modes: keep this for WYSIWYG, add one for Markdown. Would it be possible to add such a test?